runtime/databricks/automl_runtime/forecast/pmdarima/model.py

```diff
@@ -55,6 +55,15 @@ def _validate_cols(df: pd.DataFrame, required_cols: List[str]):
                 error_code=INVALID_PARAMETER_VALUE,
             )
 
+    @staticmethod
+    def _get_ds(start_ds: pd.Timestamp, periods: int, frequency: str):
```

Comment:
Let's add a docstring for this function. Also, shall we call it `_get_ds_indices`?